### PR TITLE
Define field as default_factory to specify mutable default value

### PR DESCRIPTION
**Pull request by danuriegas**

### CODE REVIEW

File: linopy/__init__.py [hunk 1]
```diff
@@ -11,4 +11,7 @@
 from linopy.io import read_netcdf
 from linopy.model import LinearExpression, Model, Variable, available_solvers
 from linopy.remote import RemoteHandler
-from linopy.version import version as __version__
+
+# from linopy.version import version as __version__
+
+__version__ = "0.0.14"
```
Comment (FabianHofmann) [inline linopy/__init__.py:17]: Could you revert this? The version specs are automatically  updated.

**Review by FabianHofmann — CHANGES_REQUESTED** [review]
Thank you @danuriegas for your contribution! Looks good. This was also tackled in #63 , but for a temporary fix we can merge this.

File: linopy/__init__.py (hunk 1 above)
Comment (danuriegas) [reply linopy/__init__.py:17]: Happy to contribute (for the first time)! I will revert this, no problem. 
Do you know why am I getting an error "ModuleNotFoundError: No module named 'linopy.version'" when importing the module?

Comment (FabianHofmann) [reply linopy/__init__.py]: I would guess this is due to `setuptools-scm` not being installed...

**Review by FabianHofmann — APPROVED** [review]

**Pull request merged**